### PR TITLE
Revamp prefix account aws

### PR DESCRIPTION
**Pull request by pipo02mix**
### Things to check/remember before submitting

- If you made content changes

    - Run `make dev` to render and proofread content changes locally.
    - Bump `last_review_date` in the front matter header if you reviewed the entire page.

### CODE REVIEW

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 1]
```diff
@@ -2,11 +2,208 @@
...
+
+The screenshot below shows an example of a service quota entry form.
+
+![Screenshot](aws-service-limits.png)
+
+Next there is the list of quotas to be modified, grouped by type:
+
+
+<!-- CAN WE MAKE SAME FOR ALL ACCOUNTS DOES NOT MATTER TYPE OF CLUSTER?-->
+
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:35]: CAN WE MAKE THE SAME FOR ALL ACCOUNTS, NOT MATTER THE TYPE OF CLUSTER?

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 2]
```diff
@@ -2,11 +2,208 @@
...
   - What do I need to do to prepare my AWS account for the cloud-native developer platform?
 ---
 
+In order to run the Giant Swarm platform in your AWS accounts, several prerequisites must be satisfied to support Cluster API Provider for AWS (CAPA).
+
+## Requirements
+
+In AWS environments you can run the management cluster and workload clusters in the same account or in separate accounts. Most of the requirements are related to configure `Identity and Access Management (IAM)` roles and service quotas.
+
+![AWS Setup Diagram](aws_onboarding.png)
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/prepare-your-provider-account/aws/_index.md:19]: @paurosello do you know if we have the source?

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 3]
```diff
@@ -2,11 +2,208 @@
...
+#### 4. Name the role {#iam-staff-role-name}
+
+Name this role:
+
+```nohighlight
+GiantSwarmAdmin
+```
+
+## Configure the organization {#configure-org}
+<!-- IS THIS STILL VALID??-->
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/prepare-your-provider-account/aws/_index.md:180]: cc/ @giantswarm/team-phoenix

**Comment by pipo02mix** [comment]
It is ready for review

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 4]
```diff
@@ -2,11 +2,216 @@
...
+
+## Step 2: Permissions
+
+There are two types of IAM roles that need to be created in the AWS account.
+
+First, the controller role which is used by the CAPA controller in the management cluster to provision all infrastructure to manage workload clusters.
+
+Second, the staff role is used by Giant Swarm engineers to access the AWS account for support purposes.
+
+Here are the mandatory steps to create the required IAM resources but we advocate to follow one of the [automated options](https://github.com/giantswarm/giantswarm-aws-account-prerequisites). There you have terraform or cloudformation templates to create the necessary IAM roles and policies. Otherwise, follow the manual steps below.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:91]:
```suggestion
Here are the mandatory steps to create the required IAM resources but we advocate to follow one of the [automated options](https://github.com/giantswarm/giantswarm-aws-account-prerequisites). There you have Terraform or CloudFormation templates to create the necessary IAM roles and policies. Otherwise, follow the manual steps below.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 5]
```diff
@@ -2,11 +2,216 @@
...
+
+### Controller permissions {#iam-capa-controller-role}
+
+#### 1. Create all policies for the capa controller {#iam-capa-controller-policies}
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:95]:
```suggestion
#### 1. Create all policies for the CAPA controller {#iam-capa-controller-policies}
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 6]
```diff
@@ -2,11 +2,216 @@
...
+
+Open the [IAM section](https://console.aws.amazon.com/iam/home) of the AWS console and go to the [Policies](https://console.aws.amazon.com/iam/home#/policies) subsection.
+
+Our platform needs different permissions to manage a variety of resources in AWS side. Here is the list of policies that need to be created. Click **Create policy** and copy the content of the policy into the JSON editor field. Hit the **Next** button and add the name of the policy matching the policy link name. Repeat this process for each policy:
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:99]:
```suggestion
Our platform needs different permissions to manage various resources on the AWS side. Here is the list of policies that need to be created. Click __Create policy__ and copy the policy content into the JSON editor field. Hit the **Next** button and add the policy name matching the policy link name. Repeat this process for each policy:
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 7]
```diff
@@ -2,11 +2,216 @@
...
+
+__Warning__: Remember to replace the `INSTALLATION_NAME` placeholder with the name of your installation when filling the policy name.
+
+![AWS IAM console: Create policy](aws-roles-create-policy.png)
+
+__Note__: All policy names contain the installation name to make easier identifying the policies and avoid conflicts with other installations running within the same account.
+
+#### 2. Create the role metadata {#iam-capa-controller-role-basic}
+
+Now move to the IAM [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection, and hit the **Create role** button. In the following screen, when asked to _Select type of trusted entity_ chose _AWS account_.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:119]:
```suggestion
Now move to the IAM [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection, and hit the __Create role__ button. In the following screen, when asked to _Select the type of trusted entity_, choose _AWS account_.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 8]
```diff
@@ -2,11 +2,216 @@
...
+
+![AWS IAM console: Create role](aws-roles-create-role.png)
+
+The **Account ID** you enter is the ID of the AWS account where management cluster is running. Could it be the current account in case you running all your infrastructure in a single account or different one if you are using separate accounts for management and workload clusters.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:123]:
```suggestion
The __Account ID__ you enter is the ID of the AWS account where the management cluster is running. Could it be the current account in case you are running all your infrastructure in a single account or a different one if you are using separate accounts for management and workload clusters?
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 9]
```diff
@@ -2,11 +2,216 @@
...
+
+It is important that the **Require external ID** and **Require MFA** options remain unchecked!
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:125]:
```suggestion
It is crucial that the __Require external ID__ and __Require MFA__ options remain unchecked!
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 10]
```diff
@@ -2,11 +2,216 @@
...
+
+Then, please go ahead and proceed to the next step.
+
+#### 3. Attach all policies to role {#iam-capa-controller-role-policy}
+
+Now you need to go through all the policies created before and attach to the node. Search for installation name, select the policies and hit the **Next** button.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:131]:
```suggestion
Now, you must go through all the policies created before and attach them to the node. Search for the installation name, select the policies and hit the __Next__ button.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 11]
```diff
@@ -2,11 +2,216 @@
...
+
+__Note__: The role name also contains the installation name to make easier identifying across other roles and avoid conflicts with other installations running within the same account.
+
+### Staff permissions {#iam-staff-role}
+
+Finally, we create an IAM role for Giant Swarm support staff to assume in order to
+access both AWS accounts. This role must have Giant Swarm's account as a trusted
+entity, and we recommend that it enforces multi-factor authentication.
+
+Giant Swarm staff require access to **all** accounts, so **the following steps must
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:153]:
```suggestion
Giant Swarm staff require access to all accounts, so the following steps must be duplicated in both the management cluster and workload cluster accounts.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 12]
```diff
@@ -2,11 +2,216 @@
...
+be duplicated in both the management cluster and workload cluster accounts**.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:154]:
```suggestion
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 13]
```diff
@@ -2,11 +2,216 @@
...
+
+#### 1. Create the admin policy {#iam-staff-policy}
+
+Go to the [Policies](https://console.aws.amazon.com/iam/home#/policies) subsection and select **Create policy** to set the admin permissions. Use the [admin JSON policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/admin-role/iam-policy.json) file as the policy content. This time, call the policy
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:158]:
```suggestion
Go to the [Policies](https://console.aws.amazon.com/iam/home#/policies) subsection and select __Create policy__ to set the admin permissions. Use the [admin JSON policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/admin-role/iam-policy.json) file as the policy content. This time, call the policy
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 14]
```diff
@@ -2,11 +2,216 @@
...
+
+```nohighlight
+GiantSwarmAdmin
+```
+
+#### 2. Create the admin role {#iam-staff-role}
+
+Enter again in the [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection of the AWS console and select **Create role**. When asked to **Select type of trusted entity** choose **AWS account**.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:166]:
```suggestion
Enter again in the [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection of the AWS console and select **Create role**. When asked to __Select type of trusted entity__ choose __AWS account__.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 15]
```diff
@@ -2,11 +2,216 @@
...
+
+- In **Account ID** enter the value `084190472784`.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:168]:
```suggestion
- In __Account ID__ enter the value `084190472784`.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 16]
```diff
@@ -2,11 +2,216 @@
...
+- **Do not** enable **Require external ID**.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:169]:
```suggestion
- Do not enable __Require external ID__.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 17]
```diff
@@ -2,11 +2,216 @@
...
+- It is strongly recommended to check the option **Require MFA** (multi factor authentication). This adds an extra authentication step for users to assume the role, which increases security.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:170]:
```suggestion
- It is strongly recommended to check the option __Require MFA__ (multi factor authentication). This adds an extra authentication step for users to assume the role, which increases security.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 18]
```diff
@@ -2,11 +2,216 @@
...
+    kind: AWSClusterControllerIdentity
+    name: default
+Where:
+
+${ROLE_NAME} is a short unique name referencing AWS account (ie: development, sandbox or staging2)
+${ROLE_ARN} is full ARN of the role created in previous step with giantswarm-aws-account-prerequisites repository.
+This CR needs to be created only once for each AWS Account. It can be then referenced in the cluster-aws repo in the value aws.awsClusterRoleIdentityName.
+```
+
+
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:212]:
```suggestion

```

File: src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md [hunk 19]
```diff
@@ -0,0 +1,77 @@
...
+    parent: overview-fleet-management-cluster-concepts
+    identifier: overview-fleet-management-cluster-concepts-multi-account
+last_review_date: 2024-06-14
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - Why should I use multi-account clusters in the Giant Swarm platform?
+---
+
+The Giant Swarm architecture distinguishes between the management cluster and workload clusters. The management cluster enables the creation and operation of workload clusters and the workload cluster(s) run your Kubernetes workloads.
```
Comment (pipo02mix) [inline src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md:16]:
```suggestion
The Giant Swarm architecture distinguishes between the management cluster and workload clusters. The management cluster enables the creation and operation of workload clusters, and the workload clusters run your Kubernetes workloads.
```

File: src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md [hunk 20]
```diff
@@ -0,0 +1,77 @@
...
+
+{{< platform_support_table aws="ga" azure="ga" >}}
+
+## How does multi-account support help? {#benefits}
+
+NEEDS TO BE REWRITTEN
+
+Both on AWS and Azure, workload cluster resources usually exist in an account (or in Azure terms: a subscription) separate from the one hosting the management cluster resources. We configure a default account to use for all workload clusters in an installation. Both accounts, the one for the management cluster and the default one for workload clusters, are under the customer's jurisdiction.
+
+**Note:** Some customers use the same account/subscription for both management cluster and workload clusters. This choice does not affect the capabilities described below.
```
Comment (pipo02mix) [inline src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md:26]:
```suggestion
**Note:** Some customers use the same account/subscription for both management cluster and workload clusters. This choice doesn't affect the capabilities described below.
```

File: src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md [hunk 21]
```diff
@@ -0,0 +1,77 @@
...
+
+- On AWS, Giant Swarms uses two separate IAM roles in order to act in the workload cluster account: one for use by automation, one for technical support staff. Details on the exact permissions required can be found in our guide on [preparing an AWS account to run Giant Swarm workload clusters]({{< relref "/vintage/getting-started/cloud-provider-accounts/vintage/aws" >}}).
+
+- On Azure, one service principal is configured for Giant Swarm, used by automation and technical support staff. Details can be found in our guide on [preparing an Azure subscription to run Giant Swarm workload clusters]({{< relref "/vintage/getting-started/cloud-provider-accounts/vintage/azure" >}}).
+
+## Additional information {#details}
+
+- Cloud provider account/subscription credentials are specified on the (Giant Swarm) **organization level**.
+
+- Cloud provider credentials are **immutable**. Once specified on an organization, cloud provider credentials cannot be modified or deleted. In order to switch to new cloud provider credentials you'll have to create a new organization and migrate to new clusters owned by that organization.
```
Comment (pipo02mix) [inline src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md:56]:
```suggestion
- Cloud provider credentials are __immutable__ Once specified in an organization, cloud provider credentials can't be modified or deleted. In order to switch to new cloud provider credentials, you'll have to create a new organization and migrate to new clusters owned by that organization.
```

File: src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md [hunk 22]
```diff
@@ -0,0 +1,77 @@
...
+
+- If an organization does not yet have provider credentials configured but already has workload clusters, these clusters are run in the default workload cluster account. Setting credentials for this organization does not affect the workload clusters created already.
```
Comment (pipo02mix) [inline src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md:58]:
```suggestion
- If an organization does not yet have provider credentials configured but already has workload clusters, these clusters are run in the default workload cluster account. Setting credentials for this organization doesn't affect the workload clusters created already.
```

File: src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md [hunk 23]
```diff
@@ -0,0 +1,77 @@
...
+
+You can then assign the credentials to your organization in several ways:
+
+- In the Giant Swarm web UI via the organization details page
+- In `gsctl` using the [`update organization set-credentials`]({{< relref "/vintage/use-the-api/gsctl/update-org-set-credentials" >}}) command
+- Via the [Giant Swarm REST API](/api/#operation/addCredentials)
+
+All workload clusters created for that organization will then use the credentials provided to the organization and will reside in the account/subscription associated with them.
+
+When inspecting details of such a cluster, or using the [`gsctl show cluster`]({{< relref "/vintage/use-the-api/gsctl/show-cluster" >}}) command, we display cloud provider details in the case the workload cluster does not reside in the default account.
```
Comment (pipo02mix) [inline src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md:77]:
```suggestion
When inspecting details of such a cluster, or using the [`gsctl show cluster`]({{< relref "/vintage/use-the-api/gsctl/show-cluster" >}}) command, we display cloud provider details in the case the workload cluster doesn't reside in the default account.
```

File: src/content/overview/fleet-management/cluster-management/cluster-concepts/private-clusters/_index.md [hunk 24]
```diff
@@ -0,0 +1,16 @@
...
+    parent: overview-fleet-management-cluster-concepts
+    identifier: overview-fleet-management-cluster-concepts-private
+last_review_date: 2024-06-14
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - What are private clusters in the Giant Swarm platform?
+---
+
+When we talk about private clusters in the Giant Swarm platform, we refer to Kubernetes clusters which has not API endpoint exposed to the public internet. This means that the cluster is only accessible from within the private network of the organization that owns the cluster.
```
Comment (pipo02mix) [inline src/content/overview/fleet-management/cluster-management/cluster-concepts/private-clusters/_index.md:16]:
```suggestion
When we talk about private clusters in the Giant Swarm platform, we refer to Kubernetes clusters, which have no API endpoint exposed to the public internet. This means that the cluster is only accessible from within the private network of the organization that owns the cluster.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 25]
```diff
@@ -2,11 +2,213 @@
...
+Here are the mandatory steps to create the required IAM resources but we advocate to follow one of the [automated options](https://github.com/giantswarm/giantswarm-aws-account-prerequisites). There you have Terraform or CloudFormation templates to create the necessary IAM roles and policies. Otherwise, follow the manual steps below.
+
+### Controller permissions {#iam-capa-controller-role}
+
+#### 1. Create all policies for the CAPA controller {#iam-capa-controller-policies}
+
+Open the [IAM section](https://console.aws.amazon.com/iam/home) of the AWS console and go to the [Policies](https://console.aws.amazon.com/iam/home#/policies) subsection.
+
+Our platform needs different permissions to manage various resources on the AWS side. Here is the list of policies that need to be created. Click __Create policy__ and copy the policy content into the JSON editor field. Hit the **Next** button and add the policy name matching the policy link name. Repeat this process for each policy:
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:99]:
```suggestion
Our platform needs different permissions to manage various resources on the AWS side. Here is the list of policies that need to be created. Click __Create policy__ and copy the policy content into the JSON editor field. Hit the `Next` button and add the policy name matching the policy link name. Repeat this process for each policy:
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 26]
```diff
@@ -2,11 +2,213 @@
...
+
+#### 2. Create the role metadata {#iam-capa-controller-role-basic}
+
+Now move to the IAM [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection, and hit the __Create role__ button. In the following screen, when asked to _Select the type of trusted entity_, choose _AWS account_.
+
+![AWS IAM console: Create role](aws-roles-create-role.png)
+
+The __Account ID__ you enter is the ID of the AWS account where the management cluster is running. Could it be the current account in case you are running all your infrastructure in a single account or a different one if you are using separate accounts for management and workload clusters?
+
+It is crucial that the __Require external ID__ and __Require MFA__ options remain unchecked!
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:125]:
```suggestion
It's crucial that the __Require external ID__ and __Require MFA__ options remain unchecked!
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 27]
```diff
@@ -2,11 +2,213 @@
...
+```nohighlight
+GiantSwarmAdmin
+```
+
+#### 2. Create the admin role {#iam-staff-role}
+
+Enter again in the [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection of the AWS console and select __Create role__ When asked to __Select type of trusted entity__ choose __AWS account__.
+
+- In __Account ID__ enter the value `084190472784`.
+- Do not enable __Require external ID__.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:168]:
```suggestion
- Don't enable __Require external ID__.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 28]
```diff
@@ -2,11 +2,213 @@
...
+- It is strongly recommended to check the option __Require MFA__ (multi factor authentication). This adds an extra authentication step for users to assume the role, which increases security.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:169]:
```suggestion
- It 's strongly recommended to check the option __Require MFA__ (multi factor authentication). This adds an extra authentication step for users to assume the role, which increases security.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 29]
```diff
@@ -2,11 +2,213 @@
...
+  roleARN: ${ROLE_ARN}
+  sourceIdentityRef:
+    kind: AWSClusterControllerIdentity
+    name: default
+Where:
+
+${ROLE_NAME} is a short unique name referencing AWS account (ie: development, sandbox or staging2)
+${ROLE_ARN} is full ARN of the role created in previous step with giantswarm-aws-account-prerequisites repository.
+This CR needs to be created only once for each AWS Account. It can be then referenced in the cluster-aws repo in the value aws.awsClusterRoleIdentityName.
+```
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:209]:
```suggestion
```

```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 30]
```diff
@@ -2,11 +2,213 @@
...
+**Note**: In case you are working with a Giant Swarm partner, you might not have access to the platform API. In that case, please provide the role ARNs values, capa controller and staff, to your partner contact.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:210]:
```suggestion
**Note**: In case you are working with a Giant Swarm partner, you might not have access to the platform API. In that case, please provide the role ARNs values, CAPA controller and staff to your partner contact.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 31]
```diff
@@ -2,11 +2,212 @@
...
+
+#### 4. Name the role {#iam-staff-role-name}
+
+Name this role:
+
+```nohighlight
+GiantSwarmAdmin
+```
+
+## Step 3: Configure the cluster role identity {#configure-cluster-role-identity}
```
Comment (AndiDog) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:183]:
````suggestion
```

Giant Swarm staff require access to all management cluster and workload cluster-related AWS accounts, so please repeat the above steps in each account.

## Step 3: Configure the cluster role identity {#configure-cluster-role-identity}
````

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 32]
```diff
@@ -2,11 +2,212 @@
...
+
+Remember that `default` AWSClusterRoleIdentity is needed but you can additional add more with specific names. (LETS INCLUDE THIS IN CAPZ TOO)
```
Comment (AndiDog) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:185]: Not reviewing this part as it's unfinished. We first have to explain what `AWSClusterRoleIdentity` is, specifically for CAPA, e.g. how the `default` one defines in which AWS account clusters will be created unless otherwise specified.

File: src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md [hunk 33]
```diff
@@ -0,0 +1,77 @@
+---
```
Comment (AndiDog) [inline src/content/overview/fleet-management/cluster-management/cluster-concepts/multi-account/_index.md:1]: Note to self: I reviewed until here

File: .vale/styles/config/vocabularies/docs/accept.txt [hunk 34]
```diff
@@ -3,13 +3,14 @@ Alertmanager
 [Aa]gent
 API[s]?
 auditable
+ARN[s]?
```
Comment (pipo02mix) [inline .vale/styles/config/vocabularies/docs/accept.txt:6]:
```suggestion
ARN[s]?
auditable
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 35]
```diff
@@ -2,11 +2,212 @@
...
+    - S3
+        - Buckets per Account: **1000**
+    - EC2 Spot Instances
+        - For every primary instance type you tend to use spot instances with, set the limit according to your needs.
+    - EC2 Instances
+        - m4.xlarge per region: **250**
+        - m4.2xlarge per region: **250**
+        - m5.2xlarge per region: **250**
+        - other instance types to be used as workers: increase accordingly
+```
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/prepare-your-provider-account/aws/_index.md:59]: I think those are fair values, it does not have impact in cost anyway

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 36]
```diff
@@ -2,11 +2,218 @@
...
 
+In this guide, you will find the necessary steps to prepare your AWS accounts to run our platform, including Cluster API Provider for AWS (CAPA).
+
+## Requirements
+
+In AWS environments, you can run the management and workload clusters in the same account or separate accounts. To help you take the decision, please read our [multi-account article]({{< relref "/overview/fleet-management/cluster-management/cluster-concepts/multi-account" >}}) where we explain the pros and cons of both approaches. Most requirements are related to configuring _Identity and Access Management (IAM)_ roles and service quotas.
+
+![AWS Setup Diagram](aws_onboarding.png)
+
+You will need access to AWS console or having AWS CLI installed and pointing to the right account in order to perform the following steps.
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:21]:
```suggestion
To perform the following steps, you will need access to the AWS console or have AWS CLI installed and pointed to the right account.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 37]
```diff
@@ -2,11 +2,212 @@
...
+clusters frequently.
+
+Every cluster needs its own VPC for security/isolation reasons and its own
+Elastic IP address for the NAT gateway.
+
+Every cluster has at least 1 Auto Scaling Group, but can contain multiple ASGs if
+multiple instance types are requested as cluster nodes. If we count 50
+clusters with up to 5 EC2 instances each, as worker nodes, we need up to 250
+ASGs. To update the ASGs in a rolling manner, we need to duplicate the ASGs
+for a short time during the update, hence the 500 Launch Configurations.
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/prepare-your-provider-account/aws/_index.md:90]: > or maybe even use default limits (I mean for CAPA)?

What are those defaults? you mean AWS defaults? those are really low

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 38]
```diff
@@ -2,11 +2,212 @@
...
+- [giantswarm-${INSTALLATION_NAME}-mc-bootstrap-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/mc-bootstrap-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-network-topology-operator-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/network-topology-operator-policy.json)
+- [giantswarm-${INSTALLATION_NAME}-resolver-rules-operator-policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/capa-controller-role/resolver-rules-operator-policy.json)
+
+__Warning__: Remember to replace the `INSTALLATION_NAME` placeholder with the name of your installation when filling the policy name.
+
+![AWS IAM console: Create policy](aws-roles-create-policy.png)
+
+__Note__: All policy names contain the installation name to make easier identifying the policies and avoid conflicts with other installations running within the same account.
+
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/prepare-your-provider-account/aws/_index.md:129]: [but we do before the manual steps](https://github.com/giantswarm/docs/pull/2218/files#diff-d8c8a411eb739312b4eade8394ff3a925eaf2319c1b87e92473c99af6fa69feaR104). We advocate for running the automation but we still offer the manual steps which also explain what exactly does

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 39]
```diff
@@ -2,11 +2,212 @@
...
+#### 2. Create the role metadata {#iam-capa-controller-role-basic}
+
+Now move to the IAM [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection, and hit the __Create role__ button. In the following screen, when asked to _Select the type of trusted entity_, choose _AWS account_.
+
+![AWS IAM console: Create role](aws-roles-create-role.png)
+
+The __Account ID__ you enter is the ID of the AWS account where the management cluster is running. Could it be the current account in case you are running all your infrastructure in a single account or a different one if you are using separate accounts for management and workload clusters?
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/prepare-your-provider-account/aws/_index.md:136]: I think it is an error of the corrector (Grammarly)

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 40]
```diff
@@ -2,11 +2,218 @@
...
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/aws/_index.md:136]:
```suggestion
The _Account ID_ you need to enter is the AWS account where the management cluster runs. In case you run all your clusters in a single account, there is no need to change it, but if your management cluster runs in a separate account, you need to use that one for `Another AWS account` option.
```

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md [hunk 41]
```diff
@@ -2,11 +2,212 @@
...
   - What do I need to do to prepare my AWS account for the cloud-native developer platform?
 ---
 
+In this guide, you will find the necessary steps to prepare your AWS accounts to run our platform. There are several requirements must be satisfied to support Cluster API Provider for AWS (CAPA).
+
+## Requirements
+
+In AWS environments, you can run the management and workload clusters in the same account or separate accounts. To help you take the decision read our [multi-account article]({{< relref "/overview/fleet-management/cluster-management/cluster-concepts/multi-account" >}}) where we explain the pros and cons of both approaches. Most requirements are related to configuring _Identity and Access Management (IAM)_ roles and service quotas.
+
+![AWS Setup Diagram](aws_onboarding.png)
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/prepare-your-provider-account/aws/_index.md:19]: WDYT?
Screenshot 2024-07-15 at 17 23 47

Source -> https://miro.com/app/board/uXjVO2Dh15w=/?moveToWidget=3458764594715037408&cot=14

File: src/content/overview/fleet-management/cluster-management/cluster-concepts/private-clusters/_index.md [hunk 42]
```diff
@@ -0,0 +1,16 @@
...
+    parent: overview-fleet-management-cluster-concepts
+    identifier: overview-fleet-management-cluster-concepts-private
+last_review_date: 2024-06-14
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - What are private clusters in the Giant Swarm platform?
+---
+
+When we talk about private clusters in the Giant Swarm platform, we refer to Kubernetes clusters, which have no API endpoint exposed to the public internet. This means that the cluster is only accessible from within the private network of the organization that owns the cluster.
```
Comment (pipo02mix) [inline src/content/overview/fleet-management/cluster-management/cluster-concepts/private-clusters/_index.md:16]: This will be extended in a different PR

File: src/content/getting-started/prepare-your-provider-account/aws/_index.md (hunk 37 above)
[the remark this answers is not in the document]
Comment (paurosello) [reply src/content/getting-started/prepare-your-provider-account/aws/_index.md:90]: we still have 1 ASG per machinepool, the change is in the number of subnets

**Review by paurosello — APPROVED** [review]
